pkg/apis/autoscaling/v1alpha1/kpa_defaults.go

```diff
@@ -28,7 +28,9 @@ func (rs *PodAutoscalerSpec) SetDefaults() {
 	if rs.ServingState == "" {
 		rs.ServingState = servingv1alpha1.RevisionServingStateActive
 	}
-	if rs.ConcurrencyModel == "" {
-		rs.ConcurrencyModel = servingv1alpha1.RevisionRequestConcurrencyModelMulti
+	// When ConcurrencyModel is specified but ContainerConcurrency
+	// is not (0), use the ConcurrencyModel value.
+	if rs.ConcurrencyModel == servingv1alpha1.RevisionRequestConcurrencyModelSingle && rs.ContainerConcurrency == 0 {
+		rs.ContainerConcurrency = 1
```

Comment:
Just to verify quickly:

The invariant is, that if `rs.ConcurrencyModel` is not set to Single, and `rs.ContainerConcurrency` is set to the default (0), we're fine?

Would it make sense to at least add that in a comment?